src/member/Sample.tsx

```diff
@@ -148,6 +164,24 @@ export const Sample = (props: {
                         </div>
                 </div>
             }
+            {
+                props.zeroday.length > 0 &&
```

Comment:
需要同時確認`props.zeroday` 是有東西的。

```suggestion
                (props.zeroday !== undefined && props.zeroday.length > 0) &&
```